src/lib/blocks.tsx

```diff
@@ -93,6 +133,23 @@ const replaceElementWithBlock = (element: HTMLElement, component: JSX.Element):
   )
 }
 
+export const renderCTABlocks = (element: HTMLElement): void => {
+  const ctaItems = element.querySelectorAll(`.${OS_RAISE_IB_CTA_CLASS}`)
+
+  ctaItems.forEach(elem => {
+    const htmlElem = elem as HTMLElement
+    const maybeCTABlock = parseCTABlock(htmlElem)
+
+    if (maybeCTABlock === null) {
+      return
+    }
+
+    htmlElem.innerHTML = ''
```

Comment:
This assignment isn't necessary since it's already handled by `replaceElementWithBlock`